custom-namespaces.yaml

```diff
@@ -9,6 +9,16 @@ operational_monitoring:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.operational_monitoring_derived.projects_v1
+combined_browser_metrics:
+  glean_app: false
+  owners:
+    - gkaberere@mozilla.com
+  pretty_name: Combined Browser Metrics
+  views:
+    projects:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.telemetry.agg_active_users
```

Comment:
The table to include would be `moz-fx-data-shared-prod.telemetry_derived.agg_active_users_v1`. Or do you normally use the corresponding view here?

Reply:
With an update to join the naming convention, the table is now `moz-fx-data-shared-prod.telemetry_derived.active_users_aggregates_v1`

Reply:
So here it looks like instead of pointing Looker at the table directly, we point it at the view:
https://github.com/mozilla/bigquery-etl/blob/c783063efc21776691de93484273d8f41197bb5d/sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

I guess this patterns is to allow for us to swap out versions of tables (in the view definition) if needed without having to make any changes in Looker.